### PR TITLE
feat(wallet): add naan wallet support

### PR DESCRIPTION
**Pull request by Jay-Tezsure**

### CODE REVIEW

File: scripts/generate-wallet-list.ts [hunk 1]
```diff
@@ -77,6 +77,15 @@ export const iosList: App[] = [
     logo: 'ios-airgap.png',
     universalLink: 'https://wallet.airgap.it',
     deepLink: 'airgap-wallet://'
+  },
+  {
+    key: 'naan_ios',
+    name: 'Naan - A Tasty Tezos Wallet',
```
Comment (pascuin) [inline scripts/generate-wallet-list.ts:83]: Please use only the wallet name and remove the additional tag line.

Comment (Jay-Tezsure) [reply scripts/generate-wallet-list.ts]: I have updated name to "Naan wallet".

**Comment by pascuin** [comment]
Thanks @Jay-Tezsure

Could you post a short video showing the Beacon integration and share links with a version where we can test it ourselves?

**Comment by Jay-Tezsure** [comment]
@pascuin
Naan Beacon integration short video:
https://user-images.githubusercontent.com/76425320/147349161-1f557a27-c41f-42a1-8190-5cfe1da113b2.mp4

App link:
https://apps.apple.com/in/app/naan-a-tasty-tezos-wallet/id1573210354

**Review by AndreasGassmann — APPROVED** [review]

**Pull request merged**